Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -56,7 +56,7 @@ configuration:
   type: 0
   section: Collect
   advanced: true
-- additionalinfo: Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances.
+- additionalinfo: "Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances.\nNote: If the test button is pressed twice, it may result in a failure and mistakenly indicate that the port is already in use."
```

Comment:
```suggestion
- additionalinfo: "Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances.\nNote: If the test button is clicked twice, it may result in a failure and mistakenly indicate that the port is already in use."
```